win/CS/HandBrakeWPF/ViewModels/MainViewModel.cs

```diff
@@ -1876,6 +1876,8 @@ public bool PresetSelect(object tag)
                         }
                     }
 
+                    this.ReGenerateAutoName();

```

Comment:
It might be worth wrapping this in so this doesn't fire in the case that the user doesn't use auto-naming. 

 if (this.userSettingService.GetUserSetting<bool>(UserSettingConstants.AutoNaming))
   {
       if (this.userSettingService.GetUserSetting<string>(UserSettingConstants.AutoNameFormat) != null)
       {
           // Regenerate
       }
   }